eksctl/opensci.jsonnet

```diff
@@ -0,0 +1,128 @@
+/*
+    This file is a jsonnet template of a eksctl's cluster configuration file,
+    that is used with the eksctl CLI to both update and initialize an AWS EKS
+    based cluster.
+
+    This file has in turn been generated from eksctl/template.jsonnet which is
+    relevant to compare with for changes over time.
+
+    To use jsonnet to generate an eksctl configuration file from this, do:
+
+        jsonnet opensci.jsonnet > opensci.eksctl.yaml
+
+    References:
+    - https://eksctl.io/usage/schema/
+*/
+local ng = import "./libsonnet/nodegroup.jsonnet";
+
+// place all cluster nodes here
+local clusterRegion = "us-west-2";
+local masterAzs = ["us-west-2a", "us-west-2b", "us-west-2c"];
+local nodeAz = "us-west-2a";
+
+// Node definitions for notebook nodes. Config here is merged
+// with our notebook node definition.
+// A `node.kubernetes.io/instance-type label is added, so pods
+// can request a particular kind of node with a nodeSelector
+local notebookNodes = [
+    { instanceType: "r5.xlarge" },
+    { instanceType: "r5.4xlarge" },
+    { instanceType: "r5.16xlarge" },
+];
+local daskNodes = [];
+
+
+{
+    apiVersion: 'eksctl.io/v1alpha5',
+    kind: 'ClusterConfig',
+    metadata+: {
+        name: "opensci",
+        region: clusterRegion,
+        version: "1.28",
+    },
```

Comment:
Nice that you push to use k8s 1.28!

This is the first time we get an EKS cluster at 1.28 - does it seem to work OK without issues? I'm confident our k8s resources etc should work in k8s 1.28 - but I'm not fully confident on not running into issues with misc EKS specific things like  `aws-ebs-csi-drivers` and such things.

Looking at https://eksctl.io/usage/schema/, it seems that k8s 1.29 is now supported, but I figure its perfectly fine if we stay on the next newest k8s version.

Based on https://infrastructure.2i2c.org/howto/upgrade-cluster/#upgrade-policy, we should go for upgrades to the latest minor version that is supported by EKS and is also considered mature based on our GKE policy - which is that its part of GKE's regular release channel - I see that 1.29 meets that criteria since [a few weeks ago](https://cloud.google.com/kubernetes-engine/docs/release-notes-regular).

Okay, this is in general off topic - but I'm glad to see we pushed up from eks 1.27 with this!!